token/cli/src/main.rs

```diff
@@ -1423,35 +1458,82 @@ async fn command_close(
             )
             .into());
         }
-        config.check_owner(&account, &source_account.owner)?;
 
-        let recipient_account = config.rpc_client.get_token_account(&recipient).await?;
-        let is_recipient_wrapped = recipient_account.map(|x| x.is_native).unwrap_or(false);
-        (is_recipient_wrapped, source_account.owner)
```

Comment:
Wow, @2501babe , just discovered that `close` has been totally broken for non-wrapped recipients (like a vanilla system account) since [this](https://github.com/solana-labs/solana-program-library/pull/3278), until your change here. Thanks for fixing!
spl-token v2.0.17 is affected, so it'll be cool to get your new cli release out :)

Reply:
oh, thank you! i tried to do a cli release recently but me and jon scuttled it due to token22 dependency version stuff between spl and monorepo, since there wasnt a pressing need to resolve at the time. ill prioritize this with jon and sam since theres regressions to fix now